src/python/grpcio/commands.py

```diff
@@ -190,8 +190,8 @@ def run(self):
                                 stderr=subprocess.STDOUT)
       except subprocess.CalledProcessError as e:
         sys.stderr.write(
-            'warning: Command:\n{}\nMessage:\n{}\nOutput:\n{}'.format(
-                command, e.message, e.output))
+            'warning: Command:\n{}\nOutput:\n{}'.format(
```

Comment:
This was an unrelated bug.  I don't think ```CalledProcessError.message``` existed for Python2.7, but I happened to exercise the code path.  I'll move this to a separate pull request.